libp2p/src/tutorials/hole_punching.rs

```diff
@@ -122,7 +122,7 @@
 //!
 //! ``` bash
 //! ## Inside the rust-libp2p repository.
-//! cargo build --example client -p libp2p-dcutr
+//! cargo build --example client
```

Comment:
Renamed to `dcutr` as far as I can tell.

Reply:
hole punching tutorial in website: [libp2p::tutorials::hole_punching - Rust](https://docs.rs/libp2p/latest/libp2p/tutorials/hole_punching/index.html#setting-up-the-listening-client)

example code still be 
`## Inside the rust-libp2p repository.
cargo build --example client -p libp2p-dcutr`

so... were the docs not updated?